lib/linguist/languages.yml

```
@@ -350,6 +350,7 @@ Apex:
   extensions:
   - ".cls"
   - ".trigger"
+  - ".apex"
```

As stated in the comment I referenced before, entries after the first still need to be in alphabetical order.

```suggestion
  - ".apex"
  - ".trigger"
```